aces/Loris
# [Test] Adding to unittests for Candidate.class.inc

## Brief summary of changes
I am adding more unit tests for [Candidate.class.inc](https://github.com/aces/Loris/blob/major/php/libraries/Candidate.class.inc) in the [CandidateTest.php](https://github.com/aces/Loris/blob/major/test/unittests/CandidateTest.php) file. I am also making changes and adding to previous tests.

#### Testing instructions 

Run "npm run tests:unit -- --filter Candidate" or check out Travis

**Note:** If you run "npm run tests:unit -- --filter Candidate" it will also run some tests from the LORIS\StudyEntitites\Candidate\Test\CandID_Test file!
